keyboards/knobgoblin/knobgoblin.c

```diff
@@ -74,22 +74,29 @@ static void render_goblin_logo(void) {
 __attribute__((weak)) void oled_task_user(void) {
 	
 	render_goblin_logo();
+    
+    oled_set_cursor(0,11);
 	
 	switch (get_highest_layer(layer_state)) {
```

Comment:
As with the JSON, but `qmk cformat keyboards/knobgoblin/knobgoblin.c` will run `clang-format` for you. (In your `keymap.c` sections where you want to keep the formatting intact, you can wrap those in `/* clang-format off */` and `/* clang-format on */`.